### DISCUSSION TITLE
Don't close drawer when saving with `⌘S`

### DISCUSSION THREAD

**Discussion by ryanleichty** [craftcms/cms#15836] [Ideas] [answered]
When editing an entry within the drawer view, I don’t expect the drawer to close when saving it with `⌘S`. I would expect the same behavior as the **Save and continue editing `⌘S`** command on entry pages where I stay in the current context.

![CleanShot 2024-10-03 at 11 55 44@2x](https://github.com/user-attachments/assets/acc5847b-8908-4497-b927-a6f3e4df1a36)

![CleanShot 2024-10-03 at 11 55 07@2x](https://github.com/user-attachments/assets/71670e22-d4a2-4a87-9169-86b348a278b8)

**Comment by mmikkel** [comment]
I disagree. On paper, being consistent about keyboard shortcuts' behaviour everywhere sounds like a good idea. However, context matters. 

Slideouts are different from dedicated element edit pages in a lot of ways (even if they've become a lot more similar recently), and as such authors generally don't _expect_ them to behave exactly the same as full edit pages.

In my experience, authors that use the Cmd+S shortcut quickly learn that it doesn't do *exactly* the same thing for dedicated edit pages and slideouts. Which I think is generally fine – Cmd+S does *save* the entry, which is the primary purpose of the shortcut. Unlike dedicated edit pages, slideouts are generally designed for _quick edits_, and in my experience having the slideout _close after saving_ (regardless of whether the "Save" button is clicked or the Cmd+S shortcut is used), seems like the expected/wanted behavior in most cases.

If slideouts' Save button ever gains multiple actions (such as the "Save and continue editing" action available in edit pages), or if they get even closer to feature parity with edit pages (e.g. if [preview targets are added to them](https://github.com/craftcms/cms/discussions/14298)) authors' expectations might change, and at that point it might make more sense to make the behavior for the Cmd+S shortcut the same across the board. I'm not sure going that direction would be the right call, but as of right now, I feel like this works as most authors expect (and wants) it to work 🙂

**Reply by ryanleichty** [reply]
With the current behavior I find myself opening nested entries in separate pages in order to avoid this so that I can quickly save the edits, check the changes on the frontend, and return to make further adjustments. I just want the behavior that already exists and that allows you to save and continue editing.

**Comment by brandonkelly** [comment, marked as answer]
Slideouts are meant for quick edits. If you want to edit an entry over multiple iterations, opening the entry in a new tab is exactly the right approach.
